Source/EasyNetQ/DI/AsyncServiceResolverScope.cs

```diff
@@ -13,11 +13,9 @@ namespace EasyNetQ.DI;
     /// </summary>
     /// <param name="serviceResolverScope">The <see cref="IServiceResolverScope"/> instance to wrap.</param>
     public AsyncServiceResolverScope(IServiceResolverScope serviceResolverScope)
-    {
-        this.serviceResolverScope = serviceResolverScope;
-    }
+        => this.serviceResolverScope = serviceResolverScope;
 
-    public IServiceResolverScope CreateScope() => new AsyncServiceResolverScope(serviceResolverScope.CreateScope());
+    public IServiceResolverScope CreateScope() => serviceResolverScope.CreateScope();
```

Comment:
Don't see any good reasons to return AsyncServiceResolverScope here, because it only could be used as `IAsyncDisposable` via `CreateAsyncScope` extension method.